recorder/requirements.txt

```diff
@@ -0,0 +1,11 @@
+helics==3.4.0
+helics-apps==3.4.0
+pydantic
+pyarrow
+scipy
+matplotlib
+numpy
+pandas
```

Comment:
scipy and matplotlib are not required here?

Reply:
already removed